Next-Notebook/Untitled.ipynb

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
<p>Is it really ok?</p>

 _Reply via <a href='https://app.reviewnb.com/slomeks/oku_notebooks/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
yes, it is